api/base.ts

```diff
@@ -18,7 +18,7 @@ import { Configuration } from "./configuration";
 // @ts-ignore
 import globalAxios, { AxiosPromise, AxiosInstance } from 'axios';
 
-export const BASE_PATH = "http://localhost".replace(/\/+$/, "");
+export const BASE_PATH = "http://192.168.0.2:8096".replace(/\/+$/, "");
```

Comment:
Maybe this is a trivial thing, but why?

Reply:
Oops, that was a local IP of mine, to test something :p

It shouldn't affect anything (we never use the BASE_PATH in the generated client), but I'll correct it anyway.

Reply:
@mrtimscampi If it's not strictly necessary I would rather have it blank instead. In case it is necessary, document it.

We don't want to have arbitrary things in our new stuff like we do right now in our current web and server :)